spec/latest/core/index.html

```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
```

Comment:
```suggestion
    <p>Hydra also allows documenting both <i>ApiDocumentation</i> and
```

https://ell.stackexchange.com/a/11197